tasks/src/main/scala/dagr/tasks/gatk/IndelRealigner.scala

```diff
@@ -34,53 +34,66 @@ import dagr.tasks.misc.DeleteFiles
 
 import scala.collection.mutable.ListBuffer
 
+object IndelRealignment {
+  // Apply method here since we can't have two constructors in IndelRealignment that take default parameters.
+  def apply(bams: Map[PathToBam, PathToBam],
+            ref: PathToFasta,
+            known: Seq[PathToVcf] = Nil,
+            intervals: Option[PathToIntervals] = None,
+            bamCompression: Option[Int] = None) = {
+    new IndelRealignment(bams=bams, ref=ref, known=known, intervals=intervals, bamCompression=bamCompression)
+  }
+}
```

Comment:
Per the comment on the apply method - you can't have two constructors with default parameters, so having this one as an apply method let's me use default parameters.